src/controllers/customer.js

```diff
@@ -43,6 +43,23 @@ const createCustomer = async (req, res) => {
   }
 }
 
+const updateCustomer = async (req, res) => {
+  const id = Number(req.params.id)
```

Comment:
What happens if req.params.id is not a number

Reply:
function isNumber(num) {
  return !isNaN(num)
}

console.log( isNumber('hello') )